types/v1/meta.go

```diff
@@ -219,6 +219,8 @@ type ObjectMeta struct {
 	ClusterName string `json:"clusterName,omitempty" protobuf:"bytes,15,opt,name=clusterName"`
 }
 
+// Labels are key/value pairs related to objects  - such as K8s and volumes -
```

Comment:
Replace  `objects  - such as K8s and volumes -`  ----> `k8s objects`

Reply:
Oh, I see. Done just now